### DISCUSSION TITLE
Bug - Comments mention notifications not sending in Scheduled Reminders Real-Time Reminders

### DISCUSSION THREAD

**Discussion by IronSean** [community/community#23874] [New to GitHub]
<p>Hi, when using the Scheduled Reminders feature and enabling Real-Time Reminders, I have the “Someone mentions you in a comment” option enabled like this:<br>
<img src="https://user-images.githubusercontent.com/1960190/181079796-dc11477f-9740-4a82-865e-7dd665fd9666.png" alt="image" data-base62-sha1="7Ere2xcoXGXbhlc9LKm1Uu94gwc" width="624" height="108"></p>
<p>However, when someone comments on an issue and tags me with <a class="mention" href="/ironsean">@IronSean</a> I am not getting github notifications. Others on my team are experiencing it as well.</p>
<p>I would expect “Someone mentions you in a comment” to apply to all comments on github (issue, code comments, PR comments, etc) but at least with issues it appears to not be working.</p>

**Comment by ettaboyle** [comment, marked as answer]
<p>Hi <a class="mention" href="/ironsean">@IronSean</a>! Our partner over in support, Anna, shared that you had submitted a ticket + reached out via the forums so I wanted to close the loop for you - and post this publicly in case anyone else encounters a similar issue. There was added content regarding a slack integration so that is addressed here as well.</p>
<p>Just as Anna shared, “At this time, <a href="https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-your-membership-in-organizations/managing-your-scheduled-reminders">Scheduled Reminders</a> is a feature targeted towards pull requests. We don’t currently have an integration that enables <span class="mention">@mentions</span> in issues to notify Slack.”</p>
<p>“However, we do have an open internal discussion to see if we can get this feature added. I can’t say if or when this feature will be implemented, but recommend keeping an eye on the <a href="https://github.blog/">GitHub Blog</a> and the <a href="https://github.com/github/roadmap">GitHub public roadmap</a> to stay up to date with new features.”</p>
<p>Also, just as an FYI, since you are an enterprise user, it might be helpful to note that there is no SLA for submissions to the forum. We don’t respond to all the posts here like a ticket queue, rather we hope that users can help support each other by answering questions, in an ideal pay-it-forward way. Our team is dedicated to sharing information on trending issues we’re seeing + driving engagement to make this a healthy, active forum full of knowledge sharing and skill-building.</p>
<p>That was a lot, but I hope it helps! Thank you for being a part of the GitHub Community :tada:<br>
-Amanda</p>

**Reply by jeiea** [reply]
I'm planning to disable slack real-time alert and use only email notification for now, just because of this lack of support. It's unfortunate that this feature hasn't been added in four years, but it's still something I'm waiting for.

**Comment by Donny10** [comment]
<p>If I could all I know is I don’t want to disrespect or take something away from your company or anything off I don’t know what enterprice I am I try to make a company and I’m having problems doing tickets trying to understand right way to do things I got sober and I’m doing the right thing that that needs to be done I need somebody to take care of my company why I get so over if there’s anybody can help me through these bug fixes and help with customer service that would be great my number is 310 817 3696 I don’t have a computer anymore long story but if anybody out there to help me sort out my problems and the agent of Law whatever I did I need to work with that I’m not crazy but I am getting sober my sobriety means a lot to me and my family too so if there’s anybody that you can fix these bugs through my system side of be great</p>

**Comment by Fgerthoffert** [comment]
In case this is useful to others, I created a small GitHub Action to act as a bridge between GitHub Notifications and an external messaging platform such as Slack.

[github.com/marketplace/actions/issues-notifications](https://github.com/marketplace/actions/issues-notifications)

The base idea is to push notifications (such as @mention in GitHub Issues) to Slack, and mark them as "done" as soon as they've been forwarded.

This is far from perfect, Scheduled Reminders working fully for Issues would be better, but that provides a solution in the interim.

Feel free to open up issues/PRs.
